nsv13/code/modules/mob/living/simple_animal/hostile/nsvsyndicate.dm

```diff
@@ -0,0 +1,139 @@
+/*

```

Comment:
This file is already under the nsv13 folder and thus doesn't need "nsv" in its title.